styles.css

```diff
@@ -1,9 +1,33 @@
-/* .main-page-form { */
+* {
+  font-family: 'Open Sans';
+}
+/* .container{
+  display: flex;
+} */
+
+.filter-starred-ideas {
+  border: 2px solid white;
+}
+.sidebar {
+  background-color: #1F1F3D;
+  color:#FFFFFF;
+  width: 300px;
+  position: fixed;
+  height: 100%;
+}
+
+.starred-idea-button {
+  background-color: #5356A4;
+  color: #E9E9F3;
+}
+
 .main-page-form {
-    font-family: 'Open Sans', Bold;
+    font-family: 'Open Sans';
+    font-weight: 100;
     color: #1F1F3D;
     background-color: #A9AAD2;
     border: 1px solid black;
+    margin-left: 300px;
 }
 
 .main-input-box {
```

Comment:
Was this deleted? Or just a placeholder for when we figure out what needs to go in here?